### PR TITLE
Move host-related modules to Managing Hosts

### PR DESCRIPTION
**Pull request by adamlazik1**
Inspect individual commits for easier review.
Disclaimer: There are no links associated with the moved modules so moving them is safe.

cc @jeremylenz 

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.5/Katello 4.7 (planned Satellite 6.13)
* [ ] Foreman 3.4/Katello 4.6 (EL8 only)
* [ ] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only)
* [ ] Foreman 3.2/Katello 4.4 on EL7 & EL8
* [ ] Foreman 3.1/Katello 4.3 on EL7 & EL8 (Satellite 6.11 EL7/8, orcharhino 6.2 on EL7/8)
* For Foreman 3.0 or older, please create a separate PR.
* We do not accept PRs for Foreman 2.3 or older.

### CODE REVIEW

File: guides/common/modules/proc_upgrading-packages-on-a-host.adoc [hunk 1]
```diff
@@ -1,5 +1,5 @@
-[id="upgrading-packages-on-a-host-in-bulk_{context}"]
-= Upgrading Packages on a Host in Bulk
+[id="upgrading-packages-on-a-host_{context}"]
+= Upgrading Packages on a Host
 
 You can upgrade packages on a host in bulk in the {ProjectWebUI}.
```
Comment (jeremylenz) [inline guides/common/modules/proc_upgrading-packages-on-a-host.adoc:4]:
```suggestion
You can upgrade packages on a host in the {ProjectWebUI}.
```

**Comment by adamlazik1** [comment]
@maximiliankolb Do you approve this?

**Comment by maximiliankolb** [comment]
Change looks good to me; please use "rebase and merge" to keep this commit as is.


Is it meant to be cherry-picked to Foreman 3.3? I thought the reason was to move "enabling/disabling repos" due to SCA which is 3.5+?

**Comment by adamlazik1** [comment]
True, my apologies. Keeping to 3.5 only.

**Comment by adamlazik1** [comment]
Merged and cherry-picked:
   bd34e2c42..97466e215  3.5 -> 3.5

**Pull request merged**